src/Models/PlayerCardModel.php

```diff
@@ -88,4 +88,30 @@ public function getAllCards(): array
 
         return $Players;
     }
+
+    public function addNewCard(
+        string $PlayerName,
+        string $Club,
+        int $Position,
+        int $Defence,
+        int $Control,
+        int $Attack,
+    ) {
+
+        $query = $this->db->prepare("INSERT INTO 
+    `PremierLeagueCards` (`PlayerName`, `Club`, `Position`, `Defence`, `Control`, `Attack`) 
+    VALUES (:PlayerName, :Club, :Position, :Defence, :Control, :Attack)");
+
+
+        $query->bindParam(':PlayerName', $PlayerName);
+        $query->bindParam(':Club', $Club);
+        $query->bindParam(':Position', $Position);
+        $query->bindParam(':Control', $Control);
+        $query->bindParam(':Attack', $Attack);
+        $query->bindParam(':Defence', $Defence);
+
+
+
+        $query->execute();
```

Comment:
You should return this - execute returns a bool to say whether or not the query worked. You can then use that return value in AddNewCard when deciding whether or not the player was added